events/test_routes.py

```diff
@@ -32,6 +32,7 @@
     'event_time': EXAMPLE_TIME,
     'extra_attribute': 'I am invalid'}
 
+# TODO(cmei4444): use for posting event information to /v1/add
```

Comment:
My feedback was that you could introduce it when you need it for the tests; it's fine to leave this here not hooked up, but it's generally easier to not add these fixtures until you're actually writing the tests.

Reply:
I'll keep this in mind!